st/clitests/rollback_spec.py

```diff
@@ -304,6 +304,17 @@ def clean_18mb_multipart():
 S3cmdTest('s3cmd can delete bucket after setting policy/acl').delete_bucket("seagatebucket").execute_test().command_is_successful()
 # ************************************************
 
+ # ************ Create bucket ************
+S3cmdTest('s3cmd can create bucket').create_bucket("seagatebucket").execute_test().command_is_successful()
+# ************ Put bucket versioning  TEST ************
+S3fiTest('s3cmd enable FI PUT KV').enable_fi("enable", "always", "motr_kv_put_fail").execute_test().command_is_successful()
+#S3cmdTest('s3cmd can not change versioning status on bucket').put_bucket_versioning("seagatebucket","Enabled").execute_test(negative_case=True).command_should_fail().command_error_should_have("InternalError")
```

Comment:
This particular test is still commented out. Should it be enabled?

Reply:
put_bucket_versioning not supported by s3cmd, so removing it